docs/panels.rst

```diff
@@ -141,6 +141,17 @@ Third-party panels
 
 If you'd like to add a panel to this list, please submit a pull request!
 
+Flame Graphs
+~~~~~~~~~~~~
+
+URL: https://gitlab.com/living180/pyflame
+
+Path: ``pyflame.djdt.panel.FlamegraphPanel``
+
+Displays a flame graph of the request, using Brendan Gregg's `flamegraph.pl
```

Comment:
I'm good with this message, but if you're wanting to call attention to your panel some people may not know what a flame graph is. You may want to include a statement that indicates what you can do with the panel.